doc/source/user_guide/plugin_creation.rst

```diff
@@ -307,7 +307,7 @@ the ``binary_path``:
         )
 
 
-For the default value, use the :py:func:`get_available_ansys_installations <ansys.tools.path.get_available_ansys_installations>`
+For the default value, use the :py:func:`get_available_ansys_installations <path.get_available_ansys_installations>`
```

Comment:
@germa89, it seems the `objects.inv` of `ansys.tools.path` has changed, which requires changing the links to its functions. Is this intentional?

Attached are the `objects.inv` files for 0.7 from https://path.tools.docs.pyansys.com/version/stable/objects.inv, and 0.6 from https://path.tools.docs.pyansys.com/version/0.6/objects.inv, both converted to JSON with `sphobjinv` + formatted.

[new.json](https://github.com/user-attachments/files/18603219/new.json)
[old.json](https://github.com/user-attachments/files/18603220/old.json)

Reply:
Converted to an issue in `ansys-tools-path`.